tinydrive/tinydrive-api/standalone.md

```diff
@@ -51,10 +74,38 @@ The URL for the selected file would be in the following format: `https://drive.t
 
 The modification date for the file in ISO 8601 format for example `2019-02-24T15:00:00Z`
 
+## `tinydrive.controls.pick`
+
+The `tinydrive.controls.pick` method enables you to pick files from {{site.cloudfilemanager}} and get the meta data of those files returned in a callback. For a complete list of available settings to pass into this method check the [Picker settings](#pickersettings) section in this page. A control object is returned which control the {{site.cloudfilemanager}} lifecycle, which it no longer does by itself.
+
+### File Picker Result Format
+
+The `tinydrive.controls.pick` method will return the selected value to the provided callback as an array of files with the following properties.
+
+#### name
```

Comment:
Would this data be better presented in a table, similar to how we show other object types in the docs?

Reply:
Possibly, but this is how it's currently presented for `tinydrive.pick`.

Reply:
Yeah, I'm wondering if we really just need to log a JIRA for someone to spend time and clean up this page.

Reply:
Sure. Log a Jira. It's out of scope for this PR.